### PR TITLE
Update final exam messages

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does:

- Updates final exam modal messaging

https://deploy-preview-646--thegymcms.netlify.app/partials/final-exam-modal-top/

https://deploy-preview-646--thegymcms.netlify.app/final-exam-modal/

### CODE REVIEW

File: final-exam-modal-top.html [hunk 1]
```diff
@@ -6,51 +6,41 @@
...
-    <p>Unfortunately, you’ve run out of chances to pass. You still deserve a lot of credit for finishing our course! Only 1 in 10 online learners actually completes a course.</p>
-
+    <h2>Darn.</h2>
+    <p>The bad news is that you didn’t pass the exam. But the good news is that we have plenty of other free courses you’ll want to check out.</p>
     <div class="survey">
-      <p>Could you please take a moment to share your thoughts with us on a quick survey? Your feedback will help shape our future courses and let us know how <em>we’re</em> doing.</p>
-      <a class="gym-button gym-button-primary" href="https://www.surveymonkey.com/s/JYJPMSS" target="_blank">
+      <p>In the meantime, please take a moment to share your thoughts with us. Your feedback will help shape our future courses!</p>
+      <a class="gym-button gym-button-primary" href="https://www.surveymonkey.com/s/JYJPMSS" target="_blank" rel="noopener">
         Take Our Survey
```
Comment (andrewpmiller) [inline final-exam-modal-top.html]: This should be "Take Survey"

File: final-exam-modal.html [hunk 2]
```diff
@@ -102,51 +102,41 @@
...
-    <p>Unfortunately, you’ve run out of chances to pass. You still deserve a lot of credit for finishing our course! Only 1 in 10 online learners actually completes a course.</p>
-
+    <h2>Darn.</h2>
+    <p>The bad news is that you didn’t pass the exam. But the good news is that we have plenty of other free courses you’ll want to check out.</p>
     <div class="survey">
-      <p>Could you please take a moment to share your thoughts with us on a quick survey? Your feedback will help shape our future courses and let us know how <em>we’re</em> doing.</p>
-      <a class="gym-button gym-button-primary" href="https://www.surveymonkey.com/s/JYJPMSS" target="_blank">
+      <p>In the meantime, please take a moment to share your thoughts with us. Your feedback will help shape our future courses!</p>
+      <a class="gym-button gym-button-primary" href="https://www.surveymonkey.com/s/JYJPMSS" target="_blank" rel="noopener">
         Take Our Survey
```
Comment (andrewpmiller) [inline final-exam-modal.html]: This should also be "Take Survey"... take out the ~"out"~ "our"... take the ~"out"~ "our" out?

**Comment by keircullenjaney** [comment]
1. Instead of a period, let's use an exclamation point at the end of the first sentence: Congratulations on passing the final exam!

2. Instead of an exclamation point at the end of the very last sentence, let's use a period: Your feedback will help shape our future courses.

**Comment by andrewpmiller** [comment]
@keircullenjaney 
> 1. Instead of a period, let's use an exclamation point at the end of the first sentence: Congratulations on passing the final exam!
> 2. Instead of an exclamation point at the end of the very last sentence, let's use a period: Your feedback will help shape our future courses.

Done!

**Pull request closed without merging**